taxcalc/functions.py

```diff
@@ -405,10 +412,9 @@ def AMED(_fica, e00200, MARS, AMED_thd, _sey, AMED_trt,
     """
 
     # ratio of income subject to AMED tax = (1 - 0.5*(FICA_mc_trt+FICA_ss_trt)
-    _amed = AMED_trt * max(0, e00200 +
-                           max(0, _sey) * (1 - 0.5 *
-                                           (FICA_mc_trt + FICA_ss_trt)) -
-                           AMED_thd[MARS - 1])
+    _amed = AMED_trt * (max(0, e00200 - AMED_thd[MARS - 1]) + max(0, max(0,
```

Comment:
The update makes a lot of sense: `max(0, e00200 - AMED_thd[MARS - 1])` corresponds to [line 7](https://www.irs.gov/pub/irs-pdf/f8959.pdf) on form 8959. And `max(0, max(0,_sey) * (1 - 0.5 * (FICA_mc_trt + FICA_ss_trt)) - max(0, AMED_thd[MARS - 1] - e00200)))` corresponds to line 8 through 13.